src/communication/azext_communication/generated/_help.py

```diff
@@ -0,0 +1,125 @@
+        text: |-
+               az communication service update --name "MyCommunicationResource" --tags newTag="newVal" \
+--resource-group "MyResourceGroup"
+"""
+
+helps['communication service delete'] = """
+    type: command
+    short-summary: "Operation to delete a CommunicationService."
+    examples:
+      - name: Delete resource
+        text: |-
+               az communication service delete --name "MyCommunicationResource" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service link-notification-hub'] = """
+    type: command
+    short-summary: "Links an Azure Notification Hub to this communication service."
+    examples:
+      - name: Link notification hub
+        text: |-
+               az communication service link-notification-hub --name "MyCommunicationResource" --connection-string \
+"Endpoint=sb://MyNamespace.servicebus.windows.net/;SharedAccessKey=abcd1234" --resource-id \
+"/subscriptions/12345/resourceGroups/MyOtherResourceGroup/providers/Microsoft.NotificationHubs/namespaces/MyNamespace/n\
+otificationHubs/MyHub" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service list-key'] = """
+    type: command
+    short-summary: "Get the access keys of the CommunicationService resource."
+    examples:
+      - name: List keys
+        text: |-
+               az communication service list-key --name "MyCommunicationResource" --resource-group "MyResourceGroup"
+"""
+
+helps['communication service regenerate-key'] = """
+    type: command
+    short-summary: "Regenerate CommunicationService access key. PrimaryKey and SecondaryKey cannot be regenerated at \
+the same time."
+    examples:
+      - name: Regenerate key
+        text: |-
+               az communication service regenerate-key --name "MyCommunicationResource" --key-type "Primary" \
+--resource-group "MyResourceGroup"
+"""
+
+helps['communication service wait'] = """
+    type: command
+    short-summary: Place the CLI in a waiting state until a condition of the communication service is met.
+    examples:
+      - name: Pause executing next line of CLI script until the communication service is successfully created.
+        text: |-
+               az communication service wait --name "MyCommunicationResource" --resource-group "MyResourceGroup" \
+--created
+      - name: Pause executing next line of CLI script until the communication service is successfully deleted.
+        text: |-
+               az communication service wait --name "MyCommunicationResource" --resource-group "MyResourceGroup" \
+--deleted
+"""
+
+helps['communication status'] = """
+    type: group
+    short-summary: communication status
+"""
+
+helps['communication status show'] = """
+    type: command
+    short-summary: "Gets the current status of an async operation."
+    examples:
+      - name: Get OperationStatus
+        text: |-
+               az communication status show --operation-id "db5f291f-284d-46e9-9152-d5c83f7c14b8" --location "westus2"
```

Comment:
If there is only one command, no need to have the command group. consider `show-status`

Reply:
done